app/assets/javascripts/sidebar.js

```diff
@@ -0,0 +1,12 @@
+
+$(document).ready(function() {
+  const optSelected = $('select.period-selection').children("option:selected").val();
+  if (optSelected === '') {
+    const url = window.location.href.split('?')[0]
+    window.history.replaceState({}, document.title, url);
+  }
```

Comment:
@santib what do you think about this? the idea is remove the query string when user select 'empty' in dropdown

Reply:
how would the query string link when selecting `empty` in dropdown without this change?

Reply:
for daily metric it is like this: 
`http://localhost:3000/projects/20/metrics/review_turnaround?metric%5Bperiod%5D=daily`
withouth that change would be like this:
`http://localhost:3000/projects/20/metrics/review_turnaround?metric%5Bperiod%5D=`

Reply:
ok I'm good leaving this 👌